Codecademy/gamut
# chore(CloseSquareIcon and FullscreenSquareIcon): add new icons

### Overview
We need the close button icon and expand icons in Gamut for our new dashboard widget. They are transparent and will be passed a color. 

Here is what the `CloseSquareIcon` looks like next to the existing  `MinimizeBarIcon ` in the monolith (with beige color)
<img width="109" alt="Screenshot 2023-06-09 at 6 00 53 PM" src="https://github.com/Codecademy/gamut/assets/12686946/2c5e86fa-9f63-4b64-9c2b-557c6c740a33">





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/TjBVb65kukF5TZRqYUn4Se/onboarding?node-id=306-11550
- [x] Related to JIRA ticket: [DISC-1212]

#### Testing Instructions
Open up the [storybook preview](https://648b4b8a969c0c2c5a0dc4f0--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon) and see the CloseSquareIcon and FullscreenSquareIcon.


#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |




[DISC-1212]: https://codecademy.atlassian.net/browse/DISC-1212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
